pkg/controller/importconfig/cluster_info_test.go

```diff
@@ -719,3 +719,31 @@ func TestGetImage(t *testing.T) {
 		t.Errorf("unexpected image: %v", newImage)
 	}
 }
+
+func TestGetBootstrapSAName(t *testing.T) {
+	cases := []struct {
+		name           string
+		clusterName    string
+		expectedSAName string
+		managedCluster *clusterv1.ManagedCluster
+	}{
+		{
+			name:           "short name",
+			clusterName:    "123456789",
+			expectedSAName: "123456789-bootstrap-sa",
+		},
+		{
+			name:           "long name",
+			clusterName:    "123456789-123456789-123456789-123456789-123456789-123456789",
+			expectedSAName: "123456789-123456789-123456789-123456789-123456789--bootstrap-sa",
```

Comment:
Are we sure here should be two `-`?

Reply:
Maybe it's `123456789-123456789-123456789-123456789-123456789-bootstrap-sa` instead of `123456789-123456789-123456789-123456789-123456789--bootstrap-sa`?

Reply:
Never mind, I got it. It will keep final length to 63.

Reply:
the rule is `<cluster-name>-bootstrap-sa`,I think it is ok to meet the rule if latest char is "-" in cluster name